packages/use-intl/test/core/createFormatter.test.tsx

```diff
@@ -253,6 +253,42 @@ describe('relativeTime', () => {
   });
 });
 
+describe('dateTimeRange', () => {
+  it('formats a date range', () => {
+    const formatter = createFormatter({
+      locale: 'en',
+      timeZone: 'Europe/Berlin'
+    });
+    expect(
+      formatter.dateTimeRange(
+        new Date(2007, 0, 10, 10, 0, 0),
+        new Date(2008, 0, 10, 11, 0, 0),
+        {
+          weekday: 'long',
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric'
+        }
+      )
+    ).toBe('Wednesday, January 10, 2007 – Thursday, January 10, 2008');
+
+    expect(
+      formatter.dateTimeRange(
+        new Date(Date.UTC(1906, 0, 10, 10, 0, 0)), // Wed, 10 Jan 1906 10:00:00 GMT
+        new Date(Date.UTC(1906, 0, 10, 11, 0, 0)), // Wed, 10 Jan 1906 11:00:00 GMT
+        {
+          year: '2-digit',
+          month: 'numeric',
+          day: 'numeric',
+          hour: 'numeric',
+          minute: 'numeric'
+        }
+      )
+    // 1 hour more given that the timezone is Europe/Berlin and the date is in UTC
```

Comment:
The linter will probably show an error here.

Reply:
what is the correct way to add comments?

Reply:
or do you want me to just delete it?

Reply:
The indent is off and therefore [the build fails](https://github.com/amannn/next-intl/actions/runs/7479432434/job/20388966214?pr=769), everything else is fine!